controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -496,9 +500,62 @@ func NewStrimzi(p *providers.Provider) (providers.ClowderProvider, error) {
 		},
 	}
 
+	createNetworkPolicies(p)
+
 	return kafkaProvider, kafkaProvider.configureBrokers()
 }
 
+func createNetworkPolicies(p *providers.Provider) error {
+	appList, err := p.Env.GetAppsInEnv(p.Ctx, p.Client)
+	if err != nil {
+		return err
+	}
+
+	namespaceSet := map[string]bool{}
+
+	np := &networking.NetworkPolicy{}
+	nn := types.NamespacedName{
+		Name:      p.Env.Name,
+		Namespace: getKafkaNamespace(p.Env),
+	}
+
+	if err := p.Cache.Create(KafkaNetworkPolicy, nn, np); err != nil {
+		return err
+	}
+
+	npFrom := []networking.NetworkPolicyPeer{}
+
+	for _, app := range appList.Items {
+		if _, ok := namespaceSet[app.Namespace]; ok {
+			continue
+		}
+
+		npFrom = append(npFrom, networking.NetworkPolicyPeer{
+			NamespaceSelector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"kubernetes.io/metadata.name": app.Namespace,
```

Comment:
does this work with the version of k8s we're running in stage/prod?

Reply:
What do you mean does this work with?